pocs/utils/social_slack.py

```diff
@@ -0,0 +1,28 @@
+import requests
+
+from pocs.utils.logger import get_root_logger
+
+
+class SocialSlack(object):
+
+    """Social Messaging sink to output to Slack."""
+
+    logger = get_root_logger()
+
+    def __init__(self, **kwargs):
+        self.web_hook = kwargs.get('webhook_url', '')
+        if self.web_hook == '':
+            raise ValueError('webhook_url parameter is not defined.')
+        else:
+            self.output_timestamp = kwargs.get('output_timestamp', False)
+
+    def send_message(self, msg, timestamp):
+        try:
+            if self.output_timestamp:
+                post_msg = '{} - {}'.format(timestamp, msg)
```

Comment:
FWIW, I think we might find it easier to consume these messages if the timestamp comes second, after msg. For example, on my phone I'd rather see timestamp truncated off than the msg.

Reply:
OK will do